recipes/dkist/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "dkist" %}
+{% set version = "1.0.0b11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dkist-{{ version }}.tar.gz
+  sha256: 54dc4a6bd60222b39f3ceb9a1e6a874b3ab2ec7466205ba48a98ff9a9ac81e04
+
+build:
+  skip: true  # [py<39]
```

This looks like it's pure Python and could be noarch:
```suggestion
  noarch: python
```